templates/flagstruct_data.F90

```diff
@@ -11,7 +11,7 @@ module flagstruct_data_mod
 {% for item in flagstruct_properties %}
     {% if item.fortran_name == "do_adiabatic_init" %}
     subroutine get_do_adiabatic_init(do_adiabatic_init_out) bind(c)
-        logical(c_bool), intent(out) :: do_adiabatic_init_out
+        logical(c_int), intent(out) :: do_adiabatic_init_out
```

Comment:
See [this comment](https://github.com/VulcanClimateModeling/fv3gfs-wrapper/pull/244/files#r597034750) for motivation here.